datahub-frontend/app/security/AuthenticationManager.java

```diff
@@ -33,7 +33,7 @@ public static void authenticateJaasUser(@Nonnull String userName, @Nonnull Strin
       LoginContext lc = new LoginContext("WHZ-Authentication", new WHZCallbackHandler(userName, password));
       lc.login();
     } catch (LoginException le) {
-      throw new AuthenticationException(le.toString(), le);
+      throw new AuthenticationException(le.toString());
```

Comment:
The correct way of setting the root exception is with initCause if we want that information to propagate

Reply:
At least for this subtype of exception, it extends javax.naming.NamingException